src/components/Menu/MenuItem/MenuItemButton/MenuItemButton.tsx

```diff
@@ -43,8 +53,13 @@ export const MenuItemButton: FC<MenuItemButtonProps> = ({
             {...rest}
         >
             {iconProps && <Icon {...iconProps} />}
-            <span className={styles.label}>{text}</span>
-            {counter && <span>{counter}</span>}
+            <div className={styles.menuItemWrapper}>
```

Comment:
can you check if accessabililty is impacted by adding divs inside the buttons

Reply:
Thanks for pointing it out. This was certainly an accessibility issue. I've converted them into span now.